packages/components/psammead-navigation/src/index.stories.jsx

```diff
@@ -27,9 +27,12 @@ import igboNavData from '../testHelpers/igbo';
 import pidginNavData from '../testHelpers/pidgin';
 import yorubaNavData from '../testHelpers/yoruba';
 import arabicNavData from '../testHelpers/arabic';
+// import { TEXT_VARIANTS } from '@bbc/psammead-storybook-helpers';
 
 import notes from '../README.md';
 
+const defaultBrandBackgroundColour = '#B80000';
```

Comment:
Should this use a constant here?

Reply:
This is used for the background colour of the brand component when shown in storybook, not by the navigation component, so I didn't feel it was relevant to the change I was making.